docs/api/migration-guides/v2-primitives.mdx

```diff
@@ -266,10 +266,9 @@ estimator = Estimator(backend, options={"resilience_level": 2})
 
 # Setting options after primitive initialization
 # This uses auto complete.
-estimator.options.optimization_level = 1
 estimator.options.default_shots = 4000
 # This does bulk update.
-estimator.options.update(default_shots=4000, optimization_level=1)
+estimator.options.update(default_shots=4000)
```

Comment:
Since this says "bulk update", I feel it should set more than 1 option. Maybe `resilience_level` or `max_execution_time` ?